packages/indexer-common/src/indexer-management/resolvers/cost-models.ts

```diff
@@ -121,22 +146,17 @@ export default {
     } catch (err) {
       throw new Error(`Invalid cost model or variables: ${err.message}`)
     }
-    const [model] = await models.CostModel.findOrBuild({
+    const [oldModel] = await models.CostModel.findOrBuild({
       where: { deployment: update.deployment },
+      order: [['createdAt', 'DESC']],
     })
-    // logger.info('Fetched current model', { current: model, update })
-    // model.set('deployment', update.deployment || model.deployment)
-    // // model.set('model', update.model || model.model)
-    // model.model = update.model || model.model
-    // logger.info('Merged models', { now: model })
-    model.deployment = update.deployment || model.deployment
-    model.model = update.model || model.model
-
-    // Update the model variables (fall back to current value if unchanged)
-    let variables = update.variables || model.variables
+    const model = models.CostModel.build()
 
+    model.deployment = update.deployment || oldModel.deployment
+    model.model = update.model || oldModel.model
```

Comment:
I think it'd be cleaner to create the new entry in one go with by giving all the values in `build`, instead of creating then updating the row column by column.
Example from the [sequelize docs](https://sequelize.org/docs/v6/core-concepts/getters-setters-virtuals/#setters):

```js
const user = User.build({
  username: 'someone',
  password: 'NotSo§tr0ngP4$SW0RD!',
});
console.log(user.password); // '7cfc84b8ea898bb72462e78b4643cfccd77e9f05678ec2ce78754147ba947acc'
console.log(user.getDataValue('password')); // '7cfc84b8ea898bb72462e78b4643cfccd77e9f05678ec2ce78754147ba947acc'
```